### ISSUE TITLE
Consul 1.0.0-beta/Nomad 0.7.0-beta: panic  interface conversion: interface {} is nil, not bool

### ISSUE THREAD

**Issue by dverbeek84** [hashicorp/nomad#3324]
### Nomad version
Nomad v0.7.0-beta1

### Operating system and Environment details
AWS
CentOS 7.2

### Issue

Nomad in agent mode panics. When running on Consul in agent mode.

Nomad in server mode on a Consul node in server mode is not a problem. This works.

### Reproduction steps

1. Install Nomad en Consul in agent mode. So make the server seperate from the node
2. Start Consul
3. Start Nomad

### Nomad Client logs (if appropriate)
```
    Loaded configuration from /etc/nomad.d/config.json
==> Starting Nomad agent...
panic: interface conversion: interface {} is nil, not bool

goroutine 1 [running]:
github.com/hashicorp/nomad/client/fingerprint.(*ConsulFingerprint).Fingerprint(0xc420396a60, 0xc42051a2c0, 0xc420494d20, 0x1eeac20, 0xc420396a60, 0x0)
	/opt/gopath/src/github.com/hashicorp/nomad/client/fingerprint/consul.go:68 +0xddd
github.com/hashicorp/nomad/client.(*Client).fingerprint(0xc42051a420, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/nomad/client/client.go:899 +0x56d
github.com/hashicorp/nomad/client.NewClient(0xc42051a2c0, 0x1eeb560, 0xc42000e020, 0x1ef36e0, 0xc42044e000, 0xc4203e4b40, 0x8, 0x15702a0, 0xc42044e000)
	/opt/gopath/src/github.com/hashicorp/nomad/client/client.go:229 +0x7a9
github.com/hashicorp/nomad/command/agent.(*Agent).setupClient(0xc420106150, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/nomad/command/agent/agent.go:503 +0x14a
github.com/hashicorp/nomad/command/agent.NewAgent(0xc420456b40, 0x1ee6860, 0xc42000c700, 0xc4203e4aa0, 0x0, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/nomad/command/agent/agent.go:90 +0x227
github.com/hashicorp/nomad/command/agent.(*Command).setupAgent(0xc420460200, 0xc420456b40, 0x1ee6860, 0xc42000c700, 0xc4203e4aa0, 0xc420277200, 0x32)
	/opt/gopath/src/github.com/hashicorp/nomad/command/agent/command.go:337 +0x96
github.com/hashicorp/nomad/command/agent.(*Command).Run(0xc420460200, 0xc420080060, 0x2, 0x2, 0x0)
	/opt/gopath/src/github.com/hashicorp/nomad/command/agent/command.go:455 +0x3ea
github.com/hashicorp/nomad/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc420456140, 0xc420456140, 0x3d, 0x1573178)
	/opt/gopath/src/github.com/hashicorp/nomad/vendor/github.com/mitchellh/cli/cli.go:255 +0x1eb
main.RunCustom(0xc420080050, 0x3, 0x3, 0xc4203ae540, 0xc420197f60)
	/opt/gopath/src/github.com/hashicorp/nomad/main.go:60 +0x1245
main.Run(0xc420080050, 0x3, 0x3, 0x1182e26)
	/opt/gopath/src/github.com/hashicorp/nomad/main.go:24 +0x56
main.main()
	/opt/gopath/src/github.com/hashicorp/nomad/main.go:20 +0x63
```

**Comment by dadgar** [comment]
@dverbeek84 Thanks for reporting. Closed as the fix was done upstream in Consul.

**Comment by dverbeek84** [comment]
@dadgar Always happy to help to make i great product 😄

**Comment by slackpad** [comment]
This should be fixed in Consul 1.0.0-beta2 - see https://groups.google.com/d/msg/consul-tool/qrdTkFVEBFQ/3hIunM91AAAJ for details.

**Comment by matthewbrahms** [comment]
Fix worked for me!  Thanks!
